extensions/stackdriver/log/exporter.cc

```diff
@@ -56,6 +56,7 @@ ExporterImpl::ExporterImpl(RootContext* root_context,
 
   // Construct grpc_service for the Stackdriver gRPC call.
   GrpcService grpc_service;
+  grpc_service.mutable_google_grpc()->set_stat_prefix("stackdriver_logging");
```

Comment:
Up on line 52 please add a failure counter one each for metrics, logging etc.

Reply:
Why are success callbacks and failure call backs lambdas. They are not capturing anything .

Reply:
I use lambda to save a function definition, and now they are capturing thing now for the metric.